core/templates/services/UpgradedServices.ts

```diff
@@ -974,6 +976,12 @@ export class UpgradedServices {
       upgradedServices['WindowRef']);
     upgradedServices['DocumentAttributeCustomizationService'] =
       new DocumentAttributeCustomizationService(upgradedServices['WindowRef']);
+    upgradedServices['EditableStoryBackendApiService'] =
+      new EditableStoryBackendApiService(
+        upgradedServices['HttpClient'],
+        upgradedServices['UrlInterpolationService']
+      );
+
```

Comment:
Actually we are trying to get rid of `UpgradedServices`. So, please don't add the service here.

Reply:
@nishantwrp yes done!